homework-7/ex-1_chat/index.html

```diff
@@ -0,0 +1,179 @@
+                   <tr>
+                        <td class="avatar">
+                            <img src="./imgs/dean-henry .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Dean Henry</h4>
+                            <span><i class="fa fa-circle" style="color: rgb(196, 137, 90);"></i>&nbsp left 3 mins ago</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/mike-thomas.jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Lee Dong Wook</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/erica-hughes .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Henry Lau</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/ginger-johnston .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Lee Jong Suk</h4>
+                            <span><i class="fa fa-circle" style="color:rgb(196, 137, 90);"></i>&nbsp left 30 mins ago</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/peyton-mckinney .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Vincent Porter</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/mike-thomas.jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Vincent Porter</h4>
+                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="avatar">
+                            <img src="./imgs/erica-hughes .jpg" alt="">
+                        </td>
+                        <td>
+                            <h4>Erica hughes</h4>
+                            <span><i class="fa fa-circle" style="color: rgb(129, 126, 109);"></i>&nbsp offline</span>
+                        </td>
+                    </tr>
```

Comment:
```suggestion
                    <tr>
                        <td class="avatar">
                            <img src="./imgs/mike-thomas.jpg" alt="">
                        </td>
                        <td>
                            <h4>Vincent Porter</h4>
                            <span><i class="fa fa-circle" style="color: green;"></i>&nbsp online</span>
                        </td>
                    </tr>
                
                    <tr>
                        <td class="avatar">
                            <img src="./imgs/erica-hughes .jpg" alt="">
                        </td>
                        <td>
                            <h4>Erica hughes</h4>
                            <span><i class="fa fa-circle" style="color: rgb(129, 126, 109);"></i>&nbsp offline</span>
                        </td>
                    </tr>
```

- Các block khác nhau thì viết cách nhau ra
- Chia layout không được dùng table